.github/workflows/actions-pipeline.yml

```diff
@@ -0,0 +1,75 @@
+name: Build
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: false
+  
+on:
+  push:
+    branches: 
+      - 'main'
+      - 'release/*'
+  pull_request:
+    branches: 
+      - 'main'
+      - 'release/*'
+      - '!/docs/'
```

Comment:
:) hopefully this works here

Reply:
according to the docs this looks valid but I haven't tested this.